style.css

```diff
@@ -170,3 +170,528 @@ nav {
 }
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  padding: 120px 0;
+  position: absolute;
+  width: 375px;
+  height: 3455px;
+  left: 0;
+  top: 657px;
+}
+
+.works_title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: 90px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border-bottom: 1px solid #212121;
+}
+
+.works_title h1 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.multi_post_container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0 24px;
+  gap: 24px;
+  width: 375px;
+  height: 568px;
+  margin-top: 50px;
+}
+
+.multi_post_img {
+  width: 327px;
+  height: 237px;
+  background: url(images/multi_post.png);
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.multi_post_stories {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+  width: 327px;
+  height: 307px;
+}
+
+.title_post {
+  width: 327px;
+  height: 36px;
+  display: flex;
+  align-items: center;
+}
+
+.title_post h2 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.multi_post_para {
+  width: 327px;
+  height: 147px;
+  border: 1px solid #fff;
+}
+
+.multi_post_para p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.tags {
+  width: 297px;
+  height: 40px;
+}
+
+.tags ul {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+}
+
+.tags ul li {
+  list-style: none;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.multi_post_stories button {
+  padding: 12px;
+  width: 124px;
+  height: 48px;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+}
+
+.works_card_container {
+  display: grid;
+  justify-content: center;
+  row-gap: 24px;
+  width: 375px;
+  height: 2436px;
+}
+
+.card {
+  width: 327px;
+  height: 386px;
+  display: flex;
+  flex-direction: column;
+}
+
+.mask_group {
+  background-image: url(images/mask_group.png);
+  background-repeat: no-repeat;
+  width: 327px;
+  height: 338px;
+}
+
+.card button {
+  width: 327px;
+  height: 48px;
+  display: flex;
+  justify-content: center;
+  padding: 12px;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
```

Comment:
- Please, kindly ensure you've added the interactivity on the button to indicate the hover, active, enabled, and disabled state on the button to improve the user experience below:

![button](https://user-images.githubusercontent.com/59397159/192068371-cb76ca99-21e5-4deb-8445-40abe07cd3ae.png)

Reply:
Hi @zieeco,

I have added all these states on the button, but implementing the button enabled is canceling the hover and active effect. What should I do now?